controllers/tasksController.js

```diff
@@ -1,4 +1,41 @@
 const tasks = require('../models/tasks')
+const userMapping = require('../utils/users')
+/**
+ * Converts the userIds entered in the array to corresponding usernames
+ * @param participantArray {array} : participants array to be updated
+ * @returns participantUsernames {array} : array of usernames of all participants
+ */
+const getParticipantUsernames = async (participantArray) => {
```

Comment:
@sanyogita14 , we should move these functions to utils/users file as these do not directly correspond to a route.

Reply:
okay